website/docs/01-man-pages.md

```diff
@@ -0,0 +1,15 @@
+---
+id: man-pages
+title: Infer Manuals
+---
+
+Here are the man pages for all the infer commands:
+
+- [infer](/static/man/infer.1.html)
+- [infer-analyze](/static/man/infer-analyze.1.html)
+- [infer-capture](/static/man/infer-capture.1.html)
+- [infer-compile](/static/man/infer-compile.1.html)
+- [infer-explore](/static/man/infer-explore.1.html)
+- [infer-report](/static/man/infer-report.1.html)
+- [infer-reportdiff](/static/man/infer-reportdiff.1.html)
+- [infer-run](/static/man/infer-run.1.html)
```

Comment:
looks like static/man/ is missing from the PR

Reply:
Fixed